src/wallet/wallet.h

```diff
@@ -994,7 +994,7 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     void ConnectScriptPubKeyManNotifiers();
 
     //! Instantiate a descriptor ScriptPubKeyMan from the WalletDescriptor and load it
-    void LoadDescriptorScriptPubKeyMan(uint256 id, WalletDescriptor& desc);
+    DescriptorScriptPubKeyMan& LoadDescriptorScriptPubKeyMan(uint256 id, WalletDescriptor& desc);
```

Comment:
In bf898af0190d:

It was removed on the last push: `LoadDescriptorScriptPubKeyMan` now returns the spkm but no function utilizes it.

Reply:
Oops, somehow that usage got dropped. Fixed.